Handle items in dc

karthikeyan5 opened the pull request:
@Ayivak , Please review as @Vigneshsekar modified your code.

Vigneshsekar committed:
Change item_code as item to avoid a loop

Vigneshsekar committed:
Add location to DC

Vigneshsekar committed:
Get boms for process from IPD BOM Mapping

Vigneshsekar committed:
Get items_to_be_sent and expected_items_in_return in DC

Vigneshsekar committed:
Merge branch 'develop' of https://github.com/aerele/apparelo into develop

Vigneshsekar committed:
Rename fields for logical flow

KaviyaPeriyasamy reviewed (changes requested):
https://github.com/aerele/apparelo/pull/91/files#diff-e7cf80170731ebae6f5fb3c6b3007cf4
The process linked to multiprocess doctype?
We need to change that?

KaviyaPeriyasamy committed:
The DC quantities are calculated and creation of apparelo process

KaviyaPeriyasamy committed:
The creation of apparelo process and process with respective secondary uom's are added

KaviyaPeriyasamy committed:
handle items with secondary uom,expected quantities in dc return items

KaviyaPeriyasamy committed:
patch file for apparelo process creation

M0hammedHaris committed:
Reload apparelo doctype

karthikeyan5 committed:
Merge branch 'develop' into handle_items_in_dc

karthikeyan5 committed:
Merge branch 'develop' of github.com:aerele/apparelo into handle_items_in_dc

karthikeyan5 reviewed (commented):
- need to add the field `secondary qty` also in DC
- wrong items returned when clicking on `Get items expected in return` button 
  - i.e. in knitting the item returned were all the 'Ironed Cloth'

karthikeyan5 committed:
fix: Get items expected in return

- structural refactor of process_based_qty fn
- adding secondary qty column in DC
- reorder and resize columns in DC items
